test/jasmine/tests/polar_test.js

```diff
@@ -1146,6 +1146,296 @@ describe('Test polar interactions:', function() {
         .then(done, done.fail);
     });
 
+    describe('dragmode === false', function() {
+        it('should not respond to drag interactions on plot area when dragmode === false', function(done) {
+            var fig = Lib.extendDeep({}, require('@mocks/polar_scatter.json'));
+
+            // to avoid dragging on hover labels
+            fig.layout.hovermode = false;
```

Comment:
Now that you added 
```js
_plot(fig)
.then(_assertBase)
.then(function() { return Plotly.relayout(gd, 'dragmode', false); })
```
let's remove 
```js
fig.layout.hovermode = false;
```
lines in new tests.
Then we should be good to merge the PR :crossed_fingers: